__tests__/app.test.js

```diff
@@ -27,4 +27,25 @@ describe('GET /api/topics', () => {
             })
         })
     })
+})
+
+describe('GET /api/articles/:article_id', () => {
+    it('returns an article object with the correct properties', () => {
+        return request(app)
+        .get(`/api/articles/4`)
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article).toMatchObject(
+                {
+                    article_id: 4,
+                    title: "Student SUES Mitch!",
+                    topic: "mitch",
+                    author: "rogersop",
+                    body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
+                    created_at: expect.any(String),
+                    votes: 0,
+                  }
+                )
+        })
```

Comment:
We are just missing some sad path tests. 
What if the id isnt found?
What is the id is invalid?